scripts/dump_meta_data.sql

```diff
@@ -0,0 +1,114 @@
+-- This script will dump relevant meta data from internal TimescaleDB tables
+-- that can help our engineers trouble shoot.
+--
+-- usage:
+-- psql [your connect flags] -d your_timescale_db < dump_meta_data.sql > dumpfile.txt
+
+\echo 'TimescaleDB meta data dump'
+\echo '<exclude_from_test> Date, git commit, and extension version can change without it being an error.'
```

Comment:
Really minor nit, but can you put `<exclude_from_test>` on its own line. Output just looks nicer/easier to read.